src/ServicePulse.Host.Tests/Owin/StaticMiddlewareTests.cs

```diff
@@ -152,7 +152,7 @@ public async Task Should_find_prefer_constants_file_on_disk_over_embedded_if_bot
                 }
             };
             await middleware.Invoke(context);
-            const long sizeOfFileOnDisk = 231; // this is the /app/js/app.constants.js file
+            const long sizeOfFileOnDisk = 232; // this is the /app/js/app.constants.js file
```

Comment:
why did this change?

Reply:
I run the prettier  on this file and it added an extra comma, see https://github.com/Particular/ServicePulse/pull/1693/files#diff-3c01f0cf02770e28d331b0ef1ee99fb21c48295bb6946ff794b6fd93b5f91374R6